.circleci/config.yml

```diff
@@ -72,7 +76,15 @@ workflows:
           filters:
             branches:
               ignore: develop
-  
+      - build_and_test:
+          name: "Double Down"
+          img: "18.04"
+          compiler: "gcc"
+          hdf5: "1.10.4"
+          moab: "master"
```

Comment:
This forces a build of MOAB `master` branch rather than using an image with a prebuilt MOAB.  Is that intentional?

Reply:
It is. There are changes to `RayHistory` in master that are necessary for double down.